lib/src/package_graph.dart

```diff
@@ -46,12 +46,11 @@ class PackageGraph {
 
   /// Returns all transitive dependencies of [package].
   ///
-  /// For the entrypoint this returns all packages in [packages], which includes
-  /// dev and override. For any other package, it ignores dev and override
-  /// dependencies.
+  /// For the root packages, this will explore the dev_dependencies if
+  /// [followDevDependenciesFromRoots] is true.
   Set<Package> transitiveDependencies(
     String package, {
-    required bool followDevDependenciesFromRoot,
+    required bool followDevDependenciesFromRoots,
```

Comment:
Isn't `package` the root? Shouldn't `followDevDependenciesFromRoot` then be singular?

Reply:
No, it is the workspace root packages.

Reply:
Ah - after talking I realize we need both conditions.
We only want to follow dev_dependencies from the `package` itself and only if it is a root package.

thanks!